docusearch/management/commands/import_documents.py

```diff
@@ -37,7 +42,6 @@ def copy_and_extract_documents(agency_directory, d):
     for document in manifest:
         doc_filename = document['document']['document_id']
         doc_path = os.path.join(date_dir, doc_filename)
-        doc_path = copy_document(doc_filename, doc_path)
```

Comment:
Looks like the copy_document function isn't being used anymore. We can remove it here or in another PR so it doesn't mess up the merge chain.